packages/eas-cli/src/graphql/queries/BranchQuery.ts

```diff
@@ -0,0 +1,33 @@
+import gql from 'graphql-tag';
+
+import { graphqlClient, withErrorHandlingAsync } from '../client';
+import { ViewBranchQuery, ViewBranchQueryVariables } from '../generated';
+
+export const BranchQuery = {
+  async getBranchByNameAsync({ appId, name }: { appId: string; name: string }) {
```

Comment:
Add a return type to the function. It helps ensure the right type is returned from the function.